flake.nix

```diff
@@ -34,11 +35,10 @@
           overlays = [ haskell-nix.overlay ];
           inherit (haskell-nix) config;
         };
-      nixpkgsFor' = system:
-        import nixpkgs {
+      nixpkgs22_05For = system:
+        import nixpkgs22_05 {
           inherit system;
           overlays = [ z3-overlay ];
```

Comment:
switched to the stable 22.05 branch for all the utilities like `fourmolu`, `eventlog2html`, etc. as well as `cabal`/`stack` in the dev shell